src/GafferAppleseed/AppleseedLight.cpp

```diff
@@ -136,6 +136,21 @@ const Gaffer::StringPlug *AppleseedLight::modelPlug() const
 	return getChild<StringPlug>( g_firstPlugIndex );
 }
 
+namespace
+{
+
+float get_min_max_value( const asf::Dictionary &inputMetadata, const std::string &key )
+{
+#if APPLESEED_VERSION >= 10800
+	const asf::Dictionary& m = inputMetadata.dictionary( key.c_str() );
+	return m.get<float>( "value" );
+#else
+	return inputMetadata.get<float>( key + "_value" ) );
```

Comment:
There's an extra closing `)` on this line that is preventing compilation for me.